plugin/dns64/README.md

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
drop the newline here

Reply:
the readme also doesn't follow the guidelines for readmes. check the other examples.

Reply:
Yea, this is just a copy paste of the upstream code.

I will add some additional commits to clean it up